js-framework-benchmark/git.json

```diff
@@ -1,6 +1,6 @@
 {
-  "url": "https://github.com/alexfmpe/js-framework-benchmark",
-  "rev": "f9dc2d0d6d603a0e4b77afeec85346cfad76c153",
-  "sha256": "1bdrgsjwbwn0zg80sg0vbb27vms817qv2fmpiz2va2y9npn9jq41",
+  "url": "https://github.com/obsidiansystems/js-framework-benchmark",
+  "rev": "c28a8c73f81970c2bd1f6444190df097c716bdcc",
```

Comment:
Only relaxes `chromedriver` to allow newer versions

Reply:
Also after switching to `yarn2nix` I'm not seeing any pulling of `chromedriver` system dep which was probably due to `npm install`ing/npm2nix related